src/lib.rs

```diff
@@ -16,20 +16,19 @@ use std::io;
 #[derive(Debug)]
 pub struct RandomAccessMemory {
   /// The length length of each buffer.
-  page_size: usize,
+  page_size: u64,
```

Comment:
At first, I didn't quite understand the change, and I've changed all `usize` items used on the function to be `u64`. After doing that, and trying to understand it better, it might not be correct, right?

Given we are using a `Vec` as our backing store, should the `page_size` should remain `usize`?

Should `length` also be `usize` to pass to the user of the API that this crate only handles 4G as memory?

Reply:
Yes, I think `page_size` should remain `usize` as it must be less than `usize::max_value()` to fit into the address space. Also, I guess in practice it will be much smaller than `usize::max_value()`.

As for `length`, it should be `u64` to handle large (sparse) storage. The current implementation doesn't seem to handle it properly, but it could (see #17).